openshift/e2e-common.sh

```diff
@@ -191,6 +191,30 @@ function run_e2e_tests(){
   local test_name=$1 
   local failed=0
 
+  # Keep this in sync with test/ha/ha.go
+  readonly REPLICAS=2
+  # TODO: increase the size of buckets when deployed chaosduck.
+  readonly BUCKETS=1
```

Comment:
This bucket option does not work with more than 1 as we don't build and deploy chaos duck image https://github.com/knative/serving/blob/master/test/config/chaosduck.yaml#L78 so it does not work with HA tests.

Reply:
Never mind. It is not related to chaosduck, but just failure of the patch command.

```
+ BUCKETS=10
+ kubectl -n knative-serving patch configmap/config-leader-election --type=merge '--patch={"data":{"buckets": "10"}}'
==== Thu Jul 16 19:09:40 PDT 2020
=======================
Error from server (BadRequest): admission webhook "config.webhook.serving.knative.dev" denied the request: validation failed: "_example" modified, you likely wanted to create an unindented configuration
```